src/endpoints/Pages.js

```diff
@@ -4,14 +4,28 @@ import { NotFoundError } from "../errors";
 import Endpoint from "./Endpoint";
 
 export default class Pages extends Endpoint {
-  info(descriptor: PageDescriptor) {
+  async info(descriptor: PageDescriptor) {
+    const latestDescriptor = await this.client.descriptors.getLatestDescriptor(
+      descriptor
+    );
+
     return this.request<Promise<Page>>({
       api: async () => {
-        const { pageId, ...fileDescriptor } = descriptor;
+        const { pageId, ...fileDescriptor } = latestDescriptor;
+        const pages = await this.list(fileDescriptor);
+        const page = pages.find(page => page.id === latestDescriptor.pageId);
```

Comment:
total nit: thoughts on using the destructured `pageId` (L14)  here?
```suggestion
        const page = pages.find(page => page.id === pageId);
```
i get that `pageId` would ideally be dropped entirely, but since that's not possible i'd almost rather use it than have an unused var hanging around. surprised eslint doesn't warn in this case.

Reply:
Good catch.